test/functional/examples/change_streams.test.js

```diff
@@ -4,7 +4,8 @@
 const setupDatabase = require('../shared').setupDatabase;
 const expect = require('chai').expect;
 
-describe('examples(change-stream):', function () {
+// Fails on MacOS on Evergreen (NODE-3713)
+describe.skip('examples(change-stream):', function () {
```

Comment:
needs unskip ticket

Reply:
Added todo comment to https://jira.mongodb.org/browse/NODE-3819